source/datasets/creativeindustries.rst

```diff
@@ -0,0 +1,17 @@
+.. carpoolinghub
+
+This dataset originates from data collected in the `DAVINCI project
+<https://davinci.bz.it/>`_, whose aims is to increase local businesses
+by improving innovation and digitalisation of services.
+
+The data consists of a list of actors that operated in the creativity
+sector in South Tyrol and is used as data source to generate a map for
+a soon to be released :ref:`Web Component <webcomponents>`.
+
+==============  ========================================================
+Output          JSON, mime-type application/json
+E-mail contact  |contact|
+API version     v2
+Swagger URL     |apiv2|
+|station|       CreativeIndustry
```

Comment:
Please always use StationType, so we do not have a confusion between what a single measurement station and a set of similar stations (aka StationTypes) are... This comment is for the whole documentation.

Reply:
Hi Peter, |station| is a placeholder and if you check the source code, it is always replaced with StationType. Check line 45 of replace.rst or online e.g., https://opendatahub.readthedocs.io/en/latest/datasets.html#rwisstation-dataset

Reply:
@stefanodavid Haha, funny, I wrote the same thing at the same time as you... see below :-)